packages/ssr-web/app/routes/not-found.ts

```diff
@@ -3,6 +3,8 @@ import { inject as service } from '@ember/service';
 import ProfileService from '@cardstack/ssr-web/services/profile';
 import RouterService from '@ember/routing/router-service';
 
+export const NotFoundError = new Error('404: Not Found');
```

Comment:
I would make this a subclass of `Error` in case there is some handler that attaches some extra data to the error.

Reply:
👍🏻 done, thanks